src/Particular.CodeRules/SuppressonIds.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Particular.CodeRules
+{
+    public static class SuppressonIds
+    {
+        public const string CancellationTokenParameterUnused = "PCRSPR0001";
```

Comment:
This came from the example shown [here](https://gist.github.com/mavasani/fcac17a9581b5c54cef8a689eeec954a#file-serializedfielddiagnosticsuppressor-cs-L17), but yes, it doesn't really fit the convention implied by PCR. And the analyzers/suppressors/etc. don't have a company name in them, since they are the "system level" analyzers, so SPR for suppression makes more sense then just S.

Following the convention implied by PCR leads me to:

- Particular Code Rules (PCR)
- Particular Suppressions (PS)

But `PS` doesn't really seem adequate. I guess you could argue that the "product name" here is Particular Code Rules, and that these are Particular Code Rules Suppressions (PCRS) but then what are the others? Particular Code Rules _Analyzers_ (PCRA)? Perhaps we should take the opportunity to come up with a better global scheme, and even rename this package (I've never liked "rules")?

Maybe:

- Package: Particular.Analyzers
- Analyzer ID's: PSCA (Particular Software Code Analyzer, following the CA convention of the "system" analyzers)
- Suppressor ID's: PSSPR (Particular Software Suppression, following the SPR convention of the example)

Or we could even be more explicit to avoid "PS" clashing with some other third party analyzer prefix:

- Analyzer ID's: PARTICULARCA (Particular Software Code Analyzer, following the CA convention of the "system" analyzers)
- Suppressor ID's: PARTICULARSPR (Particular Software Suppression, following the SPR convention of the example)

(You see what happens when you get me talking about naming?) 😁